elasticlib/Elastic.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot;
+
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.networktables.PubSubOption;
+import edu.wpi.first.networktables.PubSubOptions;
+import edu.wpi.first.networktables.StringArrayEntry;
+import edu.wpi.first.networktables.StringArrayPublisher;
+import edu.wpi.first.networktables.StringArrayTopic;
+
+public final class Elastic {
+    private static final StringArrayTopic topic = NetworkTableInstance.getDefault().getStringArrayTopic("notifications");
+    private static final StringArrayPublisher publisher = topic.publish(PubSubOption.sendAll(true));
+
+    public static void sendAlert(AlertLevel level, String title, String description) {
+        publisher.set(new String[] {level.toString(), title, description});
+    }
```

Comment:
I think it would be easier to maintain long term if it used a different format that's easier to expand on. Maybe there could be a way of publishing some sort of json map that can have different properties if we want to add more customization later on

Reply:
I think a json map would be the way to go here.